pkg/storage/bloom/v1/bloom.go

```diff
@@ -239,6 +247,16 @@ func (b *BloomBlock) BloomPageDecoder(r io.ReadSeeker, pageIdx int) (*BloomPageD
 
 	page := b.pageHeaders[pageIdx]
 
+	if page.Len > maxPageSize {
+		// TODO(chaudum): Remove print statement or replace it with a log message.
+		fmt.Fprintf(
```

Comment:
this should use a logger or bubble up to where a logger is present which we can use.